HOW/social/process/_requirements.md

```diff
@@ -0,0 +1,34 @@
+---
+Status: <DRAFT|REVIEW|REJECTED|SUPERSEDED|DEFERRED|LIVE>
+---
+## Process Specification
+
+The process specification MUST describe the workflow of the process, including the stages, steps and statuses the process goes through, and transition events that trigger status and stage changes.
+
+## Process Template Section: REVIEW
+
+After a standard is drafted, when it moves into the `REVIEW` status the following section must be added:
+
+### Steward
+
+Person or Working Group who is stewarding this HOW and is responsible for moving it through the workflow as specified by the process.
+
+## Process Template Section: REJECTED, SUPERSEDED, DEFERRED
+
+When a standard is not accepted or is superseded the following section must be added:
+
+### <status> Reason
+
+The reason this standard was not accepted or was superseded.  In the case of superseding, the content of this section MUST include which HOW, or HOW version makes this HOW obsolete.
```

Comment:
I'd question this in the SUPERSEDED case. Without building your own UI client or diffing tool, it's not that easy to see whether something other than the status reason changed.

As a user, I'd want to know that I'm reading the same text that was written when the proposal was accepted or rejected.

I think this should be metadata managed by the app and not part of the document content itself. I'd actually like the app application's code to enforce that an accepted proposal never changes again.